### PR TITLE
2596 oxfrod rpc returned type update

### PR DESCRIPTION
**Pull request by hui-an-yang**
Thank you for your contribution to Taquito.

Before submitting this PR, please make sure:

- [ ] Your code builds cleanly without any errors or warnings
- [ ] You have run the linter against the changes
- [ ] You have added unit tests (if relevant/appropriate)
- [ ] You have added integration tests (if relevant/appropriate)
- [ ] All public methods or types have TypeDoc coverage with a complete description, and ideally an @example
- [ ] You have added or updated corresponding documentation
- [ ] If relevant, you have written a first draft summary describing the change for inclusion in Release Notes. 

In this PR, please also make sure: 

- [ ] You have linked this PR to the issue by putting `closes #TICKETNUMBER` in the description box (when applicable)
- [ ] You have added a concise description on your changes
## Release Note Draft Snippet

__If relevant, please write a summary of your change that will be suitable for
inclusion in the Release Notes for the next Taquito release.__

### CODE REVIEW

File: integration-tests/known-contracts-ProxfordS.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { KnownContracts } from './known-contracts';
```
Comment (AlirezaHaghshenas) [inline integration-tests/known-contracts-ProxfordS.ts:1]: Is the renaming of this file intentional?

Comment (hui-an-yang) [reply integration-tests/known-contracts-ProxfordS.ts:1]: yes it's substring from the protocol hash which is supposedly auto-generated from `integration-tests/originate-known-contracts.ts`

File: packages/taquito-rpc/src/taquito-rpc.ts [hunk 2]
```diff
@@ -509,27 +508,28 @@ export class RpcClient implements RpcClientInterface {
...
    * @see https://tezos.gitlab.io/api/rpc.html#get-block-id
-   * @example getBlock() will default to /main/chains/block/head.
-   * @example getBlock({ block: head~2 }) will return an offset of 2 blocks.
-   * @example getBlock({ block: BL8fTiWcSxWCjiMVnDkbh6EuhqVPZzgWheJ2dqwrxYRm9AephXh~2 }) will return an offset of 2 blocks from given block hash..
+   * @example getBlock() will default to /main/chains/block/head?version=0. version=0 shows { kind: endorsement }
+   * @example getBlock({ version: 1 }) will return /main/chains/block/head?version=1. version=1 shows { kind: attestation }
+   * @example getBlock({ block: head~2 }) will return an offset of 2 blocks and default version 0.
    */
-  async getBlock({ block }: RPCOptions = defaultRPCOptions): Promise<BlockResponse> {
+  async getBlock({ block, version }: RPCOptions = defaultRPCOptions): Promise<BlockResponse> {
```
Comment (AlirezaHaghshenas) [inline packages/taquito-rpc/src/taquito-rpc.ts:520]: Should we do this in a backward-compatible way? Having a default for version?

Comment (AlirezaHaghshenas) [inline packages/taquito-rpc/src/taquito-rpc.ts:520]: There can be a design decision here: If we go with the current design, the calls to `getBlock` will need to have a version included forever. Another design might be to have an `RPC_VERSION` setting somewhere that affects this. Then the dapp can set that in one place and forget about it.

**Review by AlirezaHaghshenas — APPROVED** [review]

**Pull request merged**